sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/cosmos/properties/AzureCosmosProperties.java

```diff
@@ -78,10 +77,6 @@
      * Whether to allow for reads to go to multiple regions configured on an account of Azure Cosmos DB service.
      */
     private Boolean readRequestsFallbackEnabled;
-    /**
-     * Permission list which contains the resource tokens needed to access resources.
-     */
-    private final List<CosmosPermissionProperties> permissions = new ArrayList<>();
```

Comment:
The test properties should be removed in sync, https://github.com/Azure/azure-sdk-for-java/blob/c8a7610fcc8c36ac991c193c6bc1df52e9d09e2b/sdk/spring/spring-cloud-azure-service/src/test/java/com/azure/spring/service/implementation/cosmos/TestAzureCosmosHttpProperties.java#L41

Reply:
cool, thanks

Reply:
Will do in another PR.